apps/voting/contracts/test/mocks/VotingMock.sol

```diff
@@ -20,4 +20,16 @@ contract VotingMock is Voting {
     function isValuePct(uint256 _value, uint256 _total, uint256 _pct) external pure returns (bool) {
         return _isValuePct(_value, _total, _pct);
     }
+
+    // Create a token and a vote in the same transaction to test snapshot block valuess are correct
```

Comment:
Besides the minor typo in "valuess", "Create a token" sounds confusing to me, I'd rather say something like "Mint tokens" (and create a vote)